Multi cell database/mq adoption (#746)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_stopping-infrastructure-management-and-compute-services.adoc

[subs=+quotes]
----
ifeval::["{build}" != "downstream"]
CONTROLLER1_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@<controller-1 IP>"
# ...
endif::[]
ifeval::["{build}" == "downstream"]
CONTROLLER1_SSH="ssh -i <path_to_SSH_key> root@<controller-1 IP>"
# ...
endif::[]
# ... <1>
ifeval::["{build}" != "downstream"]
EDPM_PRIVATEKEY_PATH="~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa"
endif::[]
ifeval::["{build}" == "downstream"]
EDPM_PRIVATEKEY_PATH="<path_to_SSH_key>" <2>
endif::[]
----
+

> <1> Complete `CONTROLLER<X>_SSH` settings with SSH connection details for all controllers, including cell controllers, of the source {OpenStackPreviousInstaller} cloud.

Author: @jistr this is still valid for a single cell, just that there is no cell controllers

> <1> Complete `CONTROLLER<X>_SSH` settings with SSH connection details for all controllers, including cell controllers, of the source {OpenStackPreviousInstaller} cloud.

Reviewer: ```suggestion
<1> Complete `CONTROLLER<X>_SSH` settings with SSH connection details for all Controllers, including cell Controllers, of the source {OpenStackPreviousInstaller} cloud.
```
Should `CONTROLLER<X>_SSH`  say `CONTROLLER1_SSH`  ?

> <1> Complete `CONTROLLER<X>_SSH` settings with SSH connection details for all Controllers, including cell Controllers, of the source {OpenStackPreviousInstaller} cloud.

Reviewer: ```suggestion
<1> In the `CONTROLLER<X>_SSH` settings, provide SSH connection details for all Controller nodes, including cell Controller nodes, of the source {OpenStackPreviousInstaller} cloud.
```
